(javascript) update parameters and formatting of some references (#1912)
clerk/clerk-docs: docs/references/javascript/sign-in/second-factor.mdx

<Properties>
  - `strategy`
  - `'phone_code'`

>   The strategy to be used for second factor verification. Supported strategies are:

Reviewer: ```suggestion
  The strategy used for second factor verification. Supported strategies are:
```

  The strategy to be used for second factor verification. Possible `strategy` values are

  - `'phone_code'`: User will receive a one-time authentication code via SMS. At least one phone number should be on file for the user.

>   - `'totp'`: User must provide a 6-digit TOTP code generated by their authenticator app. The user will need to have created a TOTP secret beforehand so as to register it on their authentictor app via QR code, URI or even entering the secret itself.

Reviewer: ```suggestion
  - `'totp'`: User must provide a 6-digit TOTP code generated by their authenticator app. The user must have previously created a TOTP secret and registered it in their authenticator app using a QR code, URI, or by manually entering the secret.
```
